src/Bicep.Core.IntegrationTests/PrettyPrint/PrettyPrinterTests.cs

```diff
@@ -62,18 +64,10 @@ public void PrintProgram_AnyProgram_ShouldRoundTrip(DataSet dataSet)
             newDiagnostics.Should().HaveSameCount(diagnostics);
             newDiagnosticMessages.Should().BeEquivalentTo(diagnosticMessages);
 
-            var buffer = new StringBuilder();
-            var printVisitor = new PrintVisitor(buffer,x =>
-                // Remove newlines and whitespaces.
-                (x is Token token && token.Type == TokenType.NewLine) ||
-                (x is SyntaxTrivia trivia && trivia.Type == SyntaxTriviaType.Whitespace));
-
-            printVisitor.Visit(program);
-            string programText = buffer.ToString();
-
-            buffer.Clear();
-            printVisitor.Visit(program);
-            string formattedProgramText = buffer.ToString();
+            // Normalize formatting
+            var regex = new Regex("[\\r\\n\\s]+");
```

Comment:
This allows us to remove the test version of printVisitor. Tell me if you think it's OK.

Reply:
If they're identical, it definitely makes sense to remove the test one.